hashing/Cargo.toml

```diff
@@ -12,7 +12,7 @@ license = "Apache-2.0"
 [dependencies]
 blake2 = "0.9.0"
 base16 = "0.2.1"
-casper-types = { version = "1.5.0", path = "../types" }
+casper-types = { version = "1.5.0", path = "../types", features = ["datasize", "std"] }
```

Comment:
What changed that now we need to add these features just now? Is it something in our software or in the rustc?